app/src/routes/event/[id]/+page.svelte

```diff
@@ -66,7 +71,22 @@
 
     <div class="py-8">
       <h2 class="text-2xl font-bold pb-4">Meld deg på</h2>
-      <RegistrationForm {form} {errors} {enhance} {event} />
+      {#if $message?.success === true}
+        <Alert color="green" class="mb-6"
+          >Du har meldt deg på arrangementet! Du får en bekreftelse på <b>{$message.email}</b> hvert
+          øyeblikk.</Alert
+        >
+      {/if}
+
+      {#if $page?.status === 500}
```

Comment:
Bør sikkert gjøre dette litt annerledes på sikt, og må finne ut hva vi gjør hvis f.eks. request for påmelding går gjennom mens allergi feiler osv, per nå så er de uavhengige av hverandre